### ISSUE TITLE
It seems that you dont use Linear Evaluation Protocol?

### ISSUE THREAD

**Issue by dahuaxiya** [ildoonet/pytorch-randaugment#33]
```python
    top1, top5 = accuracy(logits, labels, topk=(1, 5))
```
the top1 accuracy doesn't seem like LEP accuracy, it seems the contrastive learning accuracy. Could you tell me where is the LEP code?please
